mcu/seproxyhal.py

```diff
@@ -187,8 +187,10 @@ def _send_packet(self, tag, data=b''):
 
     def apply_automation(self, text, x, y):
         if self.automation_server:
-            event = {"text": text.decode("ascii", "ignore"), "x": x, "y": y}
-            self.automation_server.broadcast(event)
+            lines = text.decode("ascii", "ignore").split("\n")
+            for line in lines:
+                event = {"text": line, "x": x, "y": y}
```

Comment:
I am not familiar with this code, but shouldn't `y` be updated for each line?

Reply:
Thanks for that feedback. Indeed, it's wrong. I pushed force a new commit which is a bit more complex but handles correctly that.